src/main/java/seedu/address/logic/commands/biddercommands/EditBidderCommand.java

```diff
@@ -0,0 +1,211 @@
+package seedu.address.logic.commands.biddercommands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_BIDDERS;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.CollectionUtil;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.EntityType;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.id.Id;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Phone;
+import seedu.address.model.person.bidder.Bidder;
+import seedu.address.model.tag.Tag;
+
+public class EditBidderCommand extends Command {
+
+    public static final String COMMAND_WORD = "edit-b";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the bidder identified "
+            + "by the index number used in the displayed bidder list. "
+            + "Existing values will be overwritten by the input values.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "[" + PREFIX_NAME + "NAME] "
+            + "[" + PREFIX_PHONE + "PHONE] "
+            + "[" + PREFIX_TAG + "TAG]...\n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + PREFIX_PHONE + "91234567 ";
+
+    public static final String MESSAGE_EDIT_BIDDER_SUCCESS = "Edited Bidder: %1$s";
```

Comment:
maybe have a more descriptive Success message like "Edited bidder: From [%1$s] to [%2$s]"